doc/release-process.md

```diff
@@ -0,0 +1,172 @@
+# Release Process for Antrea Operator for Kubernetes
+
+## Overview
+
+The following doc describes the release and certification process of the Antrea 
+Operator for Kubernetes project.
+The release process is combined of several steps, and each should complete 
+successfully in order to release the Operator.
+
+## Building Antrea-derived resources
+
+Some the resources within the Antrea Operator repository are derived from the 
+content of the Antrea repository, and therefore should be updated upon release 
+of Antrea versions.\
+These resources can be generated automatically by running a 
+Makefile target:
+
+```bash
+make antrea-resources 
+```
+
+This Makefile target generates the following resources:
+- `antrea-manifest/antrea.yml`: a template file containing Antrea resources and 
+derived from Antrea
+- `config/rbac/role.yaml`: contains the role all the roles which are required 
```

Comment:
`contains the role all the roles`, mistype here?

Reply:
Indeed, I'll fix. Thanks for catching that :)